liblwgeom/lwgeom_geos.c

```diff
@@ -101,23 +101,18 @@ ptarray_from_GEOSCoordSeq(const GEOSCoordSequence* cs, char want3d)
 
 /* Return an LWGEOM from a Geometry */
 LWGEOM*
-GEOS2LWGEOM(const GEOSGeometry* geom, char want3d)
+GEOS2LWGEOM(const GEOSGeometry* geom, uint32_t want3d)
```

Comment:
Why change the type of the boolean?

Reply:
it's `int` in functions and `char` here.